src/AmplifyGeofenceControl/index.ts

```diff
@@ -136,15 +136,16 @@ export class AmplifyGeofenceControl {
     }
     const feature = this._amplifyDraw.get(this._editingGeofenceId);
 
+    const idToSave = geofenceId || this._editingGeofenceId;
     const response = await Geo.saveGeofences({
-      geofenceId: geofenceId || this._editingGeofenceId,
+      geofenceId: idToSave,
```

Comment:
Just curious: This is presumably where the issue with the geofences being saved was throwing? What was happening here?

Reply:
Naw this was just something I added for convenience sake. The actual issue was that on the API side you must save geofences that have coordinates that are in counter clockwise order. Originally I was saving Polygonal geofences in clockwise orders. This wasn't an issue with circular geofences since they were already respecting the counter clockwise order

Reply:
Awesome, thanks for the heads up!